frontend/packages/console-plugin-shared/src/utils/datetime.ts

```diff
@@ -0,0 +1,41 @@
+import * as _ from 'lodash';
+
+// Conversions between units and milliseconds
+const s = 1000;
+const m = s * 60;
+const h = m * 60;
+const d = h * 24;
+const w = d * 7;
+const units = { w, d, h, m, s };
+
+// Formats a duration in milliseconds like "1h 10m"
```

Comment:
I don't think we're generating docs for console-plugin-shared yet, but it would be good to add ts-doc here for when we do.

Reply:
👍